mytoyota/models/endpoints/trip.py

```diff
@@ -0,0 +1,95 @@
+from pydantic import BaseModel, Field
+from datetime import date, datetime
+from uuid import UUID
+
+
+class _Pagination(BaseModel):
+    currentPage: int
+    limit: int
+    offset: int
+    pageCount: int
+    previousOffset: int
+    totalCount: int
+
+
+class _SortedBy(BaseModel):
+    field: str
+    order: str
+
+
+class _MetaData(BaseModel):
+    pagination: _Pagination
+    sortedBy: list[_SortedBy]
+
+
+class _Scores(BaseModel):
+    acceleration: int = Field(ge=0, le=100)
+    advice: int = Field(ge=0, le=100, default=0)
+    braking: int = Field(ge=0, le=100)
+    constantSpeed: int = Field(ge=0, le=100, default=0)
+    global_: int | None = Field(ge=0, le=100, alias="global", default=None)
+
+
+class _Summary(BaseModel):
+    countries: list[str]
+    duration: int
+    durationHighway: int
+    durationIdle: int
+    durationOverspeed: int
+    fuelConsumption: float
+    length: int
+    lengthHighway: int
+    lengthOverspeed: int
+    maxSpeed: float
+
+
+class _HDC(BaseModel):
+    # Depending on
+    #    car not being EV
+    #    car being ev only
+    #    car being hybrid but only used ev/fuel
+    chargeDist: int = Field(0)
+    chargeTime: int = Field(0)
+    ecoDist: int = Field(0)
+    ecoTime: int = Field(0)
+    evDistance: int = Field(0)
+    evTime: int = Field(0)
+    powerDist: int = Field(0)
+    powerTime: int = Field(0)
+
+
+class _MonthSummary(BaseModel):
+    hdc: _HDC | None = Field(default=None)  # Only available on EV cars
```

Comment:
A more pythonic way for optional values is in my opinion:

```python
from typing import Optional

    hdc: Optional[_HDC] = None  # Only available on EV cars

```
btw you don't need the `Field` object to define default Values.